<ui version="4.0" >
 <class>rangedlg</class>
 <widget class="QWidget" name="rangedlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>340</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Set Verse Range</string>
  </property>
  <widget class="QPushButton" name="cancel" >
   <property name="geometry" >
    <rect>
     <x>262</x>
     <y>130</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>130</y>
     <width>50</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="fromBox" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>From:</string>
   </property>
   <widget class="QSpinBox" name="verse1" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>20</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
   </widget>
   <widget class="QComboBox" name="book1" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>180</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="chapter1" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>20</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ToBox" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>60</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>To:</string>
   </property>
   <widget class="QSpinBox" name="verse2" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>20</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
   </widget>
   <widget class="QComboBox" name="book2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>180</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="chapter2" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>20</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>rangedlg</receiver>
   <slot>rangedlg_ok()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>rangedlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>book1</sender>
   <signal>activated(int)</signal>
   <receiver>rangedlg</receiver>
   <slot>book1selected(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chapter1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rangedlg</receiver>
   <slot>chapter1selected(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>book2</sender>
   <signal>activated(int)</signal>
   <receiver>rangedlg</receiver>
   <slot>book2selected(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chapter2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rangedlg</receiver>
   <slot>chapter2selected(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
